qa/workunits/rados/test_rados_tool.sh

```diff
@@ -455,38 +460,24 @@ test_cleanup() {
 
 function test_append()
 {
-  # rados append test:
-  # replicated pool
-  $CEPH_TOOL osd pool create rados_append 100 100 replicated
+  cleanup
+
   # create object
   touch ./rados_append_null
-  $RADOS_TOOL -p rados_append append rados_append_obj ./rados_append_null
-  $RADOS_TOOL -p rados_append get rados_append_obj ./rados_append_0_out
-  orig_size=`ls -l ./rados_append_null | awk -F ' '  '{print $5}'`
-  $RADOS_TOOL -p rados_append get rados_append_obj ./rados_append_0_out
-  orig_size=`ls -l ./rados_append_null | awk -F ' '  '{print $5}'`
-  read_size=`ls -l ./rados_append_0_out | awk -F ' '  '{print $5}'`
-  if [ $orig_size -ne $read_size ];
-  then
-    die "Create Failed!"
-  fi
+  $RADOS_TOOL -p $POOL append $OBJ ./rados_append_null
+  $RADOS_TOOL -p $POOL get $OBJ ./rados_append_0_out
+  cmp ./rados_append_null ./rados_append_0_out
+  cleanup
 
   # append 4k, total size 4k
   dd if=/dev/zero of=./rados_append_4k bs=4k count=1
-  $RADOS_TOOL -p rados_append append rados_append_obj ./rados_append_4k
-  $RADOS_TOOL -p rados_append get rados_append_obj ./rados_append_4k_out
-  orig_size=`ls -l ./rados_append_4k  | awk -F ' '  '{print $5}'`
-  read_size=`ls -l ./rados_append_4k_out | awk -F ' '  '{print $5}'`
-  if [ $orig_size -ne $read_size ];
-  then
-    die "Append failed expecting $orig_size read $read_size"
-  fi
+  $RADOS_TOOL -p $POOL append $OBJ ./rados_append_4k
```

Comment:
this changes the behavior of this test. prior to this change, we

1. create an empty object
1. append 4k "0" blob to it
1. compare it with the 4k blob

after this change

1. append 4k "0" blob to a non-exisitent object
1. compare it with the 4k blob